pkg/render/fluentd.go

```diff
@@ -458,7 +458,8 @@ func (c *fluentdComponent) packetCaptureApiRoleBinding() *rbacv1.RoleBinding {
 // managerDeployment creates a deployment for the Tigera Secure manager component.
 func (c *fluentdComponent) daemonset() *appsv1.DaemonSet {
 	var terminationGracePeriod int64 = 0
-	maxUnavailable := intstr.FromInt(1)
```

Comment:
Only concern is that this could break all of the nodes at once if a bad configuration is rolled out for some reason, whereas previously I think we would have stopped after rolling out one node and the other nodes would continue to function..

That said, I think it's probably better to just be faster in this case so I'm good with this.